test/uk/gov/hmrc/servicedependencies/persistence/SlugInfoRepositorySpec.scala

```diff
@@ -121,6 +123,7 @@ class SlugInfoRepositorySpec
 
```

Comment:
I see the MongoDB persistence/serialization classes work equally well with Java time and Joda.  We've got use of Joda time elsewhere to represent dates, which might cause some future confusion if we need to compare incompatible date/time representations.  I wonder if we should think about standardizing on Java 8 time instead of joda time.

This is a wider consideration, probably not worth addressing in this PR.